spec/models/repository_spec.rb

```diff
@@ -18,7 +18,7 @@
 
   describe 'scopes' do
     describe '.only_mirrored' do
```

Comment:
Should we also change the description here?

Reply:
I just saw this myself and pushed a change.

Reply:
:+1: